x-pack/plugins/observability_solution/observability/server/plugin.ts

```diff
@@ -20,7 +20,7 @@ import {
   PluginInitializerContext,
 } from '@kbn/core/server';
 import { LogsExplorerLocatorParams, LOGS_EXPLORER_LOCATOR_ID } from '@kbn/deeplinks-observability';
-import { PluginSetupContract as FeaturesSetup } from '@kbn/features-plugin/server';
+import { FeaturesPluginSetup as FeaturesSetup } from '@kbn/features-plugin/server';
```

Comment:
nit: Shall we remove the renaming then, now that there is no name conflict anymore?

Reply:
Yeah absolutely (sorry I mostly regexp-replaced, so I missed those ones)

Reply:
done in https://github.com/elastic/kibana/pull/187944/commits/f36c6a2a521ca906f5b7cec43167aba1a2396520